src/features/cli/ReverseShellConnection.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import styled from "@emotion/styled";
+import { ExpandMore } from "@mui/icons-material";
+import {
+  Card, CardContent, Collapse, CardHeader, IconButton, TextField, Button,
+} from "@mui/material";
+import { Box } from "@mui/system";
+import { useCallback } from "react";
```

Comment:
Move to top of file - I try to logically group imports together:
1. React related imports
2. MUI related imports
3. Local imports

```
import React, {
  useCallback,
} from "react";
```